app/src/main/java/com/example/datarsd1/forecastapp/Utils.java

```diff
@@ -0,0 +1,76 @@
+package com.example.datarsd1.forecastapp;
+
+import android.content.Context;
+import android.content.Intent;
+import android.location.Address;
+import android.location.Geocoder;
+import android.os.AsyncTask;
+
+import org.json.JSONObject;
+
+import java.io.BufferedInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Adhiraj Datar on 3/6/2017.
+ */
+
+public class Utils
+{
+
+    Context context;
+
+    public Utils(Context context)
+    {
+        this.context = context;
+    }
+
+    public LatLong geocodeName(String name)
+    {
+        List<LatLong> l = new ArrayList<LatLong>();
+        if(Geocoder.isPresent()){
+            try {
+                Geocoder gc = new Geocoder(context);
+                List<Address> addresses= gc.getFromLocationName(name, 5); // get the found Address Objects
+
```

Comment:
This is interesting, but it should output the specific location. I put poop and it worked, and I couldn't find out where that is. How is someone in San Jose, Costa Rica going to know they got the right city's weather?